testcases/lib/tst_net.sh

```diff
@@ -128,6 +128,9 @@ tst_net_require_ipv6()
 
 init_ltp_netspace()
 {
+	PATH="$PWD/../../testcases/kernel/containers/share/:$PATH"
+	tst_res TINFO "PATH='$PATH'"
```

Comment:
This is not acceptable. The path should go to the test itself (`tst_net.sh` should be either proper installed, then it expect `ns_clone` and other binaries to be properly in `/opt/ltp/testcases/bin` and `$PATH` set to it. Or, if it's used for a development, user is required to set the correct `$PATH` already.)
The reason why it should be in `tst_rhost_run.sh` is to allow testing instead of `make install`. But user still needs to run make in required directories to have required binaries at least compiled (if not installed).

I recommend you that you to check commands here. I meant to check commands with `tst_require_cmds` shell function in `tst_net.sh`.